internal/lsp/server.go

```diff
@@ -692,8 +617,9 @@ func (l *LanguageServer) handleInitialize(
 	l.clientIdentifier = clients.DetermineClientIdentifier(params.ClientInfo.Name)
 
 	if l.clientIdentifier == clients.IdentifierGeneric {
-		l.log(
-			"Unable to match client identifier for initializing client, using generic functionality: " + params.ClientInfo.Name,
+		l.logError(
+			fmt.Errorf("unable to match client identifier for initializing client, using generic functionality: %s",
+				params.ClientInfo.Name),
```

Comment:
So it's an error, but it's not so bad that we cannot go on with some default? Is it an error, then? 🤔